nvi-evaluator/src/test/java/no/sikt/nva/nvi/evaluator/EvaluateNviNviCandidateHandlerTest.java

```diff
@@ -285,7 +320,13 @@ void shouldThrowExceptionIfFileDoesntExist() throws IOException {
 
     @Test
     void shouldCreateNonCandidateEventWhenZeroNviInstitutions() throws IOException {
-        when(uriRetriever.fetchResponse(any(), any())).thenReturn(Optional.of(notFoundResponse));
+        var cristinOrgResponse = createResponse(200, getHardCodedCristinOrgResponse());
+        var cristinOrgSubUnit = URI.create("https://api.dev.nva.aws.unit.no/cristin/organization/194.64.20.0");
```

Comment:
Convert cristin/organization and customer/cristinId to constants, since they are used in every test?